test/api/v3/integration/groups/POST-groups_groupId_join.js

```diff
@@ -12,23 +23,62 @@ describe('POST /group/:groupId/join', () => {
         name: 'Test Guild',
         type: 'guild',
```

Comment:
Let's make this explicitly a private guild. Right now private is the default, but let's not count on that for this test.